packages/app/src/components/DBRowTable.tsx

```diff
@@ -273,7 +274,7 @@ const SqlModal = ({
       ) : isLoadingSql ? (
         <div className="text-center my-2">
           <div className="spin-animate d-inline-block me-2">
-            <i className="bi bi-arrow-repeat" />
+            <IconRefresh size={14} />
```

Comment:
Here, and maybe elsewhere when animating this icon, it looks a little wonky


https://github.com/user-attachments/assets/97fdeea9-885d-4068-9bf7-0d3392146e5f